mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -64,8 +64,9 @@ private[ann] trait Layer extends Serializable {
    * @return the layer model
    */
   def createModel(initialWeights: BDV[Double]): LayerModel
+
   /**
-   * Returns the instance of the layer with random generated weights
+   * Returns the instance of the layer with random generated weights.
```

Comment:
Yes, I see, but it just says "returns the layer model", which is what the text above already says in more detail. I didn't see a point in repeating it